### PR TITLE
Improved player_start conversion

### PR DESCRIPTION
**Pull request by Mac-tf2**

### CODE REVIEW

**Comment by Panzerhandschuh** [comment]
Does this only work the first time the player spawns?

**Comment by Mac-tf2** [comment]
yeah, but so did the old version. Should be fixed with rios savestate system.

File: BSPConvert.Lib/Source/EntityConverter.cs [hunk 1]
```diff
@@ -403,109 +403,18 @@ private void ConvertPlayerStart(Entity playerStart)
...
-					var ammo = CreateTargetGiveAmmo(ammoName, playerStart.Origin, count);
-					sourceEntities.Add(ammo);
-				}
-				else if (target.ClassName.StartsWith("item_"))
-				{
-					var itemName = GetMomentumItemName(target.ClassName);
-					var item = CreateTargetGiveItem(itemName, playerStart.Origin, target["count"]);
-					sourceEntities.Add(item);
-				}
+				ConvertEntityTargetsRecursive(logicAuto, logicAuto, "OnMapSpawn", 0, new HashSet<Entity>());
```
Comment (Panzerhandschuh) [inline BSPConvert.Lib/Source/EntityConverter.cs]: I think you can just pass `playerStart` as the target instead of assigning the logic_auto's target.

Comment (Mac-tf2) [reply BSPConvert.Lib/Source/EntityConverter.cs]: yeah that works

**Review by Panzerhandschuh — APPROVED** [review]

**Pull request merged**